RestApiForGroup2/README.md

```diff
@@ -0,0 +1,14 @@
+Server which responds to REST API requests from the group which mines the unstructured data
+URIs supported:
+http://127.0.0.1:3000/entities/ - HTTP method GET fetches all the entries in the database
+http://127.0.0.1:3000/entities/entity/:id  HTTP method GET fetches entry in the database whose entity id matches id
+http://127.0.0.1:3000/entities/entity/:id  HTTP method PUT updates entry in the database whose entity id matches id
+http://127.0.0.1:3000/entities/entity/ method POST fetches adds new entry in the database 
+http://127.0.0.1:3000/entities/entity/:id  HTTP method DELETE deletes entry in the database whose entity id matches id
+
+
+Group 2 shall use these endpoints to filter the unstructured data .
+
+Client.js file describes the HTTP request that Group2 should generate
```

Comment:
Please use some styling to improve readability: https://guides.github.com/features/mastering-markdown/